### ISSUE TITLE
Add annotations to serialization of BEL Statements

### ISSUE THREAD

**Issue by ncatlett** [OpenBEL/bel.rb#28]
BEL Statement.annotations are not currently included in the to_bel output of Statement objects. 

There is a possibility for Annotation.name clash (and potentially Annotation duplicates) within this array which need to be handled during serialization of Annotations for a Statement. The expected behavior would be to handle multiple Annotations with the same name and different values by serializing the values as an array. For example, if a Statement has both the Annotations Disease:"Diabetes" and Disease:"Obesity", it should serialize as "SET Disease = {Diabetes, Obesity}". Duplicate Annotations should be serialized only once.

Ideally, the Annotations will be serialized in a defined order (i.e., 1st Citation, EvidenceText, then all others in alphabetical order)

**Comment by abargnesi** [comment]
This has been superseded by the BEL::Model::Evidence model though grouping and sorting the experiment context/metadata is still incomplete.
